src/status_im/ui/screens/subs.cljs

```diff
@@ -54,3 +54,7 @@
   :<- [:sync-state]
   (fn [sync-state]
     (#{:pending :in-progress} sync-state)))
+
+(reg-sub :view-params
+  (fn [db [_ view-id]]
+    (get-in db [:screen-params (or view-id (:view-id db))])))
```

Comment:
This is little bit confusing, our "convention" (well it's meant to be) is to name the subscription `:get-app-db-prop-name`, so it would be `:get-screen-params` instead of the `:view-params`.